### ISSUE TITLE
Peer certificate cannot be authenticated with given CA certificates for "https://security.symfony.com/check_lock"

### ISSUE THREAD

**Issue by merlin334** [sensiolabs/security-checker#165]
When i run:
php security-checker security:check composer.lock
I get the following error:
```javascript
Peer certificate cannot be authenticated with given CA certificates for "https://security.symfony.com/check_lock".
In CurlResponse.php line 264:
Peer certificate cannot be authenticated with given CA certificates for "https://security.symfony.com/check_lock".

security:check [--format FORMAT] [--end-point END-POINT] [--timeout TIMEOUT] [--token TOKEN] [--] [<lockfile>]
```

![image](https://user-images.githubusercontent.com/9083576/60094603-3349ce00-9755-11e9-8c53-ae50d61f8381.png)

- Tried it on 2 different desktops, 2 laptops and vmware all with windows 10.
- Tried with 3 different internet providers, one being tethering from mobile 4G network Vodafone.
- Disabled antivirus, and firewall.

Any idea how to fix this?
Thank you.

**Comment by nicolas-grekas** [comment]
Fixed already in Symfony HttpClient 4.3-dev, will be released as part of 4.3.2 .

**Comment by fabpot** [comment]
Let's close then.
